### DISCUSSION TITLE
Node.js version is not recognized correctly

### DISCUSSION THREAD

**Discussion by Espo88** [vercel/next.js#59546] [Help]
### Summary

I recently updated to Next.js 14.0.4 and when I try `npm run dev` I receive: `You are using Node.js 18.7.0. For Next.js, Node.js version >= v18.17.0 is required.` even when I check the node.js version with `node -v` it gives me `v20.10.0`. What should I do? Any help is appreciated, thanks.

**Comment by pybern** [comment]
Can you post your `package.json`?

**Comment by PaskalevStoyan** [comment]
This email was not delivered due to system error. I discovered it and forwarded to you. issue has been fixed since then

Best Regards

Daniel ***@***.***>
Sr. Systems Engineer

***@***.***<http://www.progress.com/>

PROGRESS SOFTWARE CORPORATION
15 Wayside Rd, Suite 400
Burlington, MA 01803 USA

DIRECT  1 781 207 9256

WWW.PROGRESS.COM<http://www.progress.com/>

From: Espo88 ***@***.***>
Sent: Tuesday, December 12, 2023 4:55 PM
To: vercel/next.js ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [vercel/next.js] Node.js version is not recognized correctly (Discussion #59546)

Summary

I recently updated to Next.js 14.0.4 and when I try npm run dev I receive: You are using Node.js 18.7.0. For Next.js, Node.js version >= v18.17.0 is required. even when I check the node.js version with node -v it gives me v20.10.0. What should I do? Any help is appreciated, thanks.

Additional information

No response

Example

No response

**Comment by Espo88** [comment]
I updated Macos and now it works.

**Comment by d-mmmy** [comment]
Hi all! 
I had a run in with this issue too. I had somehow stupidly at some point years back, installed node via `npm -g` into my root. **There was a node with an old exe in my `c:/users/<yourname>/node_modules`.** 

I just shift deleted the whole modules folder and now it runs perfectly.
